Dribbblish/dribbblish.js

```diff
@@ -124,6 +113,37 @@ waitForElement([
     updateVariable();
 });
 
+// allow resizing of the buddy feed
+waitForElement([".Root__right-sidebar .LayoutResizer__input, .Root__right-sidebar .LayoutResizer__resize-bar input"], ([resizer]) => {
+    const observer = new MutationObserver(updateVariable);
+    observer.observe(resizer, { attributes: true, attributeFilter: ["value"] });
+    function updateVariable() {
+        let value = resizer.value;
+        if (value == 320) {
+            value = 72;

```

Comment:
What do these number mean actually? Do you meant to make the buddy feed sidebar collapse all the way after it reaches a certain threshold?

Reply:
yeah, thats exactly what it does, since the minimum width of the sidebar that spotify allows the user to set is 320, so i used that as the threshold.